jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyPlugin.groovy

```diff
@@ -41,30 +40,30 @@ class JRubyPlugin implements Plugin<Project> {
             JRubyExec.updateJRubyDependencies(project)
         }
 
-        project.task('jrubyPrepareGems', type: JRubyPrepareGems) {
+        project.task('jrubyPrepareGems', type: JRubyPrepare) {
```

Comment:
jrubyPrepareGems just needs to be an empty task with one action added and a dependency added. No need to make it a JRubyPrepare task.

```groovy
Task jrpg = project.task.create 'jrubyPrepareGems'
jrpg.dependsOn 'jrubyPrepare'
jrpg << { logger.info "'jrubyPrepareGems' is deprecated and will be removed in a future version. Use 'jrubyPrepare' instead." }
```